cmake/app/boilerplate.cmake

```diff
@@ -517,6 +523,12 @@ set(KERNEL_EXE_NAME   ${KERNEL_NAME}.exe)
 set(KERNEL_STAT_NAME  ${KERNEL_NAME}.stat)
 set(KERNEL_STRIP_NAME ${KERNEL_NAME}.strip)
 
+define_property(GLOBAL PROPERTY ${IMAGE}KERNEL_NAME
+  BRIEF_DOCS "Name (KERNEL_NAME) for the ${IMAGE} image."
+  FULL_DOCS "To be used to access e.g. this image's hex file."
+  )
+set_property(GLOBAL PROPERTY ${IMAGE}KERNEL_NAME ${KERNEL_NAME})
```

Comment:
This is sane as well.

I would await introducing it until there was a user though.

In case this isn't quite what you are looking for. E.g. it might turn out that

${${IMAGE}logical_name_for_zephyr_elf} is what you actually need.

Reply:
name -> target?
`${${IMAGE}logical_target_for_zephyr_elf}`

Reply:
I see what you mean.

Reply:
Removed